hq/app/schedule/unrecognised/IamUnrecognisedUserJob.scala

```diff
@@ -33,24 +43,39 @@ class IamUnrecognisedUserJob(cacheService: CacheService, snsClient: AmazonSNSAsy
       logger.info(s"Running scheduled job: $description")
     }
 
-    val dummyJanusData: JanusData = ???
-
-    val janusUsers: Seq[String] = getJanusUsernames(dummyJanusData)
-    val allCredsReports = cacheService.getAllCredentials
+    val result = for {
+      config <- getConfigProperties(config)
+      client <- s3Clients.get(config.securityAccount)
+      s3Object <- getS3Object(client, config.janusUserBucket, config.janusDataFileKey)
```

Comment:
This s3GetObject call fails with 
```
The authorization header is malformed; the region 'us-east-1' is wrong; expecting 'eu-west-1' (Service: Amazon S3; Status Code: 400; Error Code: AuthorizationHeaderMalformed;)
```

It works if s3Clients in AppComponents is initialised with the eu-west-1 region
```
  private val s3Clients = AWS.s3Clients(configuration, Regions.EU_WEST_1)
```
This is not necessarily the right solution, but there's something going on with the region!